.gitlab/kitchen_deploy/kitchen_deploy.yml

```
@@ -245,6 +245,36 @@ deploy_rpm_testing-a7_arm64:
     - set +x
     - echo "$RPM_SIGNING_PASSPHRASE" | rpm-s3 --verbose --visibility public-read -c "https://s3.amazonaws.com" -b $RPM_TESTING_S3_BUCKET -p "testing/pipeline-$DD_PIPELINE_ID/7/aarch64/" -a "aarch64" --sign --metadata-signing-key $RPM_GPG_KEY_ID $OMNIBUS_PACKAGE_DIR/datadog-*-7.*aarch64.rpm
 
+.deploy_rpm_testing-u7:
+  stage: kitchen_deploy
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/gitlab_agent_deploy:$DATADOG_AGENT_BUILDERS
+  tags: ["arch:amd64"]
+  rules:
+    - !reference [.except_no_tests_no_deploy]
```

```suggestion
    - !reference [.except_no_tests_no_deploy]
    - !reference [.except_mergequeue]
    - when: on_success
```
The `except_mergeque` is probably not necessary but it would align all rules together. I'll do some cleanup later